pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>kg.apc</groupId>
     <artifactId>jmeter-plugins-manager</artifactId>
-    <version>0.15</version>
+    <version>0.17</version>
```

Comment:
The version choice is made on release, it is not meant to be changed by every PR. Just for the future info...